src/widgets/create-exhibition/model/useExhibitionForm.ts

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { convertAddressToCoordinates } from './api';
+
+export const useExhibitionForm = () => {
+  const [title, setTitle] = useState<string>('');
+  const [textAreaContent, setTextAreaContent] = useState('');
+  const [img, setImg] = useState<string | null>(null);
+  const [address, setAddress] = useState('');
+
+  const handleConvertAddress = async () => {
+    if (address) {
+      const coordinates = await convertAddressToCoordinates(address);
+      if (coordinates) {
+        console.log('좌표:', coordinates);
+      } else {
+        console.log('좌표 변환에 실패했습니다.');
+      }
+    } else {
+      console.log('주소를 입력해주세요.');
+    }
+  };
```

Comment:
단순 콘솔만 출력할 거라면 좀 더 간결하게 다음처럼 사용할 수 있을 것 같은데 어떤가요?
주관적인 것이니 참고만 해주세용
```suggestion
  const handleConvertAddress = async () => {
    if (!adress) {
      console.log('주소를 입력해주세요.');
      return;
  }
  
  const coordinates = await convertAddressToCoordinates(adress);
  console.log(coordinates ? `좌표: ${coordinates}` : 좌표 변환에 실패했습니다.);
```
이렇게 adress가 없다면 ealry return 하는 것도 좋아보입니다.